### PR TITLE
all: replace deprecated "ioutil" package with their "os" and "io" counterparts

### PR DESCRIPTION
**Pull request by abhinavnair**
As of Go 1.16, the same functionality is now provided by package io or
package os, and those implementations should be preferred in new code.

So replacing all usage of `ioutil` pkg with `io` & `os`.

### CODE REVIEW

File: contrib/google.golang.org/api/api_test.go [hunk 1]
```diff
@@ -6,15 +6,15 @@
...
-	"io/ioutil"
+	"io"
 	"net/http"
 	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
-	books "google.golang.org/api/books/v1"
-	civicinfo "google.golang.org/api/civicinfo/v2"
-	urlshortener "google.golang.org/api/urlshortener/v1"
```
Comment (gbbr) [inline contrib/google.golang.org/api/api_test.go:17]: This is unrelated to your change. Can you please revert this part of the PR?

File: ddtrace/example_test.go [hunk 2]
```diff
@@ -6,10 +6,10 @@
 package ddtrace_test
 
 import (
-	"io/ioutil"
 	"log"
+	"os"
 
-	opentracing "github.com/opentracing/opentracing-go"
```
Comment (gbbr) [inline ddtrace/example_test.go:12]: Same here.

File: contrib/google.golang.org/api/api_test.go (hunk 1 above)
Comment (abhinavnair) [reply contrib/google.golang.org/api/api_test.go:17]: Ok. Let me revert those changes and stick to specifically the ioutil changes in this PR 🙂

**Review by gbbr — APPROVED** [review]
Great! Thanks!

File: internal/appsec/config.go [hunk 3]
```diff
@@ -7,7 +7,6 @@ package appsec
 
 import (
 	"fmt"
```
Comment (nsrip-dd) [inline internal/appsec/config.go:9]:
```suggestion
	"fmt"
	"io"
```

File: internal/appsec/config_test.go [hunk 4]
```diff
@@ -9,7 +9,6 @@
 package appsec
 
 import (
-	"io/ioutil"
 	"os"
```
Comment (nsrip-dd) [inline internal/appsec/config_test.go:12]:
```suggestion
	"io"
	"os"
```

File: internal/appsec/waf_test.go [hunk 5]
```diff
@@ -9,7 +9,6 @@
 package appsec_test
 
 import (
-	"io/ioutil"
 	"net/http"
```
Comment (nsrip-dd) [inline internal/appsec/waf_test.go:12]:
```suggestion
	"io"
	"net/http"
```

**Review by nsrip-dd — CHANGES_REQUESTED** [review]
Looks like there are a few missing `"io"` imports.

**Review by nsrip-dd — APPROVED** [review]

**Review by Hellzy — APPROVED** [review]
LGTM, thanks 👍

**Review by ajgajg1134 — APPROVED** [review]
